src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -214,8 +214,11 @@ internal class ConnectionManager : IConnectionManager {
 
         private bool IsValidLocalConnection(string name, string path) {
             try {
-                var info = new RInterpreterInfo(name, path);
-                return info.VerifyInstallation();
+                Uri uri;
+                if (Uri.TryCreate(path, UriKind.Absolute, out uri) && uri.IsFile) {
```

Comment:
I'm not sure how good it is to show progress bar right on top of the VS that has just started. We can have an indication on a status bar.